vs_scanwatch/Service1.cs

```diff
@@ -0,0 +1,325 @@
+        public static string SetRootDirectory(string rootFolder)
+        {
+            if (Directory.Exists(rootFolder))
+            {
+                //This does nothing because it gets called before the logger gets initialized, so it's commented out and left for reference
+                //Log.Information($"Override folder found on drive. Root folder set to \"{rootFolder}\".");
+                return rootFolder;
+            }
+
+            string defaultFolder = @"C:\Scan\";
+
+            if (!Directory.Exists(defaultFolder))
+            {
+                System.IO.Directory.CreateDirectory(defaultFolder);
+            }
+
+            //This does nothing because it gets called before the logger gets initialized, so it's commented out and left for reference
+            //Log.Information($"Override folder not found on drive. Root folder set to default value \"{defaultFolder}\".");
+            return defaultFolder;
+        }
+
+        //This method checks to see if a subfolder with the selected date name already exists inside the specified root folder and if not, it creates that folder
+        public static string SetOutputDirectory(string rootFolder, string dateFolder)
+        {
+            string outputFolder = Path.Combine(rootFolder, dateFolder)+"\\";
+            System.IO.Directory.CreateDirectory(outputFolder);
+            return outputFolder;
+        }
+
+
+        //This method checks to see if a valid date override has been entered and returns the name of the output folder
+        public static string GetFolderName(string dateTimeOverride)
+        {
+            Log.Information($"Root folder set to \"{FolderPath.rootDirectoryPath}\"."); //This gets logged here because the root folder gets selected before the logger gets initialized
+            DateTime validDate;
+            //Checks if the provided override is a valid date and if not returns current date in Belgrade
+            if (false == DateTime.TryParseExact(dateTimeOverride, "yyyyMMdd", new CultureInfo("sr-Latn-RS"), DateTimeStyles.AdjustToUniversal, out validDate))
+            {
+                return SetFolderNameToBelgradeTimeDate();
+            }
+
+            string output = validDate.ToString("yyyyMMdd");
+
+            Log.Information($"Valid override date detected. Destination folder set to \"{output}\".");
+            return output;
+        }
+
+        //This method gets called if no valid date override has been provided and generates the folder name according to the current date in Belgrade
+        public static string SetFolderNameToBelgradeTimeDate()
+        {
+            string output = TimeZoneInfo.ConvertTimeFromUtc(DateTime.UtcNow, TimeZoneInfo.FindSystemTimeZoneById("Central Europe Standard Time")).ToString("yyyyMMdd");
+            Log.Information($"No valid override date detected. Destination folder set to \"{output}\".");
```

Comment:
This whole method perhaps should not exist, because whole logic is just one line with log info.